docs/content/guides/rows/row-sorting.md

```diff
@@ -1,176 +1,609 @@
 ---
 id: 6o0zftmc
-title: Row sorting
-metaTitle: Row sorting - JavaScript Data Grid | Handsontable
-description: Sort your data in ascending, descending, or a custom order, across one or multiple rows of the data grid.
+title: Rows sorting
+metaTitle: Rows sorting - JavaScript Data Grid | Handsontable
+description: Sort rows alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.
 permalink: /row-sorting
 canonicalUrl: /row-sorting
+tags:
+  - row sorting
+  - column sorting
+  - columnSorting
+  - multicolumn sorting
+  - multi-column sorting
+  - multiColumnSorting
+  - row ordering
+  - order by
+  - sort ascending
+  - sort descending
+  - highest to lowest
+  - unordered data
+  - sortEmptyCells
+  - data sorting
+  - sort data
+  - sort rows
+  - sort columns
+  - sorting
 react:
   id: h4jfevxj
-  metaTitle: Row sorting - React Data Grid | Handsontable
+  metaTitle: Rows sorting - React Data Grid | Handsontable
 searchCategory: Guides
 ---
 
-# Row sorting
+# Rows sorting
 
-Sort your data in ascending, descending, or a custom order, across one or multiple rows of the grid.
+Sort data alphabetically or numerically, in ascending, descending or a custom order, across one or multiple columns.
 
 [[toc]]
 
 ## Overview
 
-The [`ColumnSorting`](@/api/columnSorting.md) plugin works as a middleware between the data source and the Handsontable rendering module.
+With sorting, you can easily rearrange rows of data, based on the values in specified columns.
```

Comment:
"Selected columns" may sound more natural, but it's ambiguous. In the context of HoT, "selected columns" may mean [cell selection](https://handsontable.com/docs/react-data-grid/selection/). It could suggest that when you select a column (i.e., click on that column's header), the data gets sorted, which is not true (you need to click on the column's label/header).

I'd keep the word "selected" as a reserved, technical term, and use it consistently only in the context of cell/row/column selection per se.